src/index.ts

```diff
@@ -25,7 +26,8 @@ export const parsers: Record<string, Parser> = {
     svelte: {
         parse: (text) => {
             try {
-                return <ASTNode>{ ...require(`svelte/compiler`).parse(text), __isRoot: true };
+                // @ts-ignore
```

Comment:
should add a comment why this is necessary. if unfixable should use `@ts-expect-error`